xcdat/regridder/base.py

```diff
@@ -32,22 +33,22 @@ def preserve_bounds(
         Target Dataset with preserved bounds.
     """
     try:
-        lat_bnds = source_grid.cf.get_bounds("lat")
+        lat_bnds = source_grid.bounds.get_bounds("Y")
     except KeyError:
         pass
     else:
         target[lat_bnds.name] = lat_bnds.copy()
 
     try:
-        lon_bnds = source_grid.cf.get_bounds("lon")
+        lon_bnds = source_grid.bounds.get_bounds("X")
     except KeyError:
         pass
     else:
         target[lon_bnds.name] = lon_bnds.copy()
 
     for dim_name in source.cf.axes:
         try:
-            source_bnds = source.cf.get_bounds(dim_name)
+            source_bnds = source.bounds.get_bounds(dim_name)
```

Comment:
I replaced the API call to `.cf.get_bounds()` with `bounds.get_bounds()` because our API checks if the coordinate variable exists (e.g., "lat") and its `"bounds"` attr is set to the bounds data var (e.g., "lat_bnds").

If both of those conditions aren't meant, it will raise an error.